src/views/virtualmachines/details/tabs/diagnostic/VirtualMachineLogViewer/hooks/useVirtualMachineLogData.ts

```
@@ -1,3 +1,4 @@
+// eslint-disable-next-line unicorn/prefer-node-protocol
```

ESLint

[error] Definition for rule 'unicorn/prefer-node-protocol' was not found.

(unicorn/prefer-node-protocol)